src/features/events/layout/EventLayout.tsx

```diff
@@ -49,115 +50,120 @@ const EventLayout: React.FC<EventLayoutProps> = ({
   }
 
   return (
-    <TabbedLayout
-      actionButtons={
-        <ZUIFuture future={eventFuture}>
-          {(data) => {
-            return <EventActionButtons event={data} />;
-          }}
-        </ZUIFuture>
-      }
-      baseHref={getEventUrl(eventFuture.data)}
-      defaultTab="/"
-      subtitle={
-        <Box alignItems="center" display="flex">
-          <Box marginRight={1}>
-            <EventStatusChip state={eventState} />
-          </Box>
-          <ZUIFutures
-            futures={{
-              currentEvent: eventFuture,
-              types: eventTypes,
+    <>
+      <Head>
+        <title> {eventFuture.data?.title} </title>
```

Comment:
Ah I believe I forgot to tell you. If there is no title, the title should use the activity title, and if that's not available, it should use the noTitle message.

Here is an example of how this is implemented:
https://github.com/zetkin/app.zetkin.org/blob/c279be9988118c47fe8d1db5e6339914aefb16d2/src/features/events/components/EventPopper/SingleEvent.tsx#L132